projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-site-connection-notice.tsx

```diff
@@ -17,14 +16,8 @@ type RedBubbleAlerts = Window[ 'myJetpackInitialState' ][ 'redBubbleAlerts' ];
 const useSiteConnectionNotice = ( redBubbleAlerts: RedBubbleAlerts ) => {
 	const { recordEvent } = useAnalytics();
 	const { setNotice, resetNotice } = useContext( NoticeContext );
-	const { apiRoot, apiNonce } = getMyJetpackWindowRestState();
-	const { siteIsRegistering, handleRegisterSite } = useConnection( {
+	const { handleRegisterSite, siteIsRegistering } = useMyJetpackConnection( {
```

Comment:
I like updating the `useMyJetpackConnection` to simplify the usage of this. Should we open up a followup PR to replace all other instances of `useConnection` or was this just for files that need specific properties?

Reply:
Created maintenance item for this - will follow up with the PR shortly!

Reply:
Awesome!